src/applications/break/DaGdBreakController.php

```diff
@@ -1,13 +1,13 @@
 <?php
 
-final class DaGdBreakController extends DaGdBaseClass {
-  public function render() {
+final class DaGdBreakController extends DaGdController {
+  public function execute(DaGdResponse $response) {
     $environment = DaGdConfig::get('general.environment');
     if ($environment != 'development') {
-      error400('This page is disabled in the production environment.');
-      return false;
+      return $this->error(400,
+        'This page is disabled in the production environment.')->execute($response);
```

Comment:
```suggestion
      return $this
        ->error(400, 'This page is disabled in the production environment.')
        ->execute($response);
```

Just for consistency.

Reply:
Is there a specific linter that would make these multi-lining decisions for this repo?